source/install/trouble_mysql.rst

```diff
@@ -0,0 +1,268 @@
+.. _mysql-installation-troubleshooting:
+
+MySQL Installation Troubleshooting
+----------------------------------
+
+Before you can run the Mattermost server, you must first install and
+configure a database. You can then start Mattermost by navigating to
+``/opt/mattermost/bin`` and entering the command ``sudo -u mattermost ./platform``.
+If the Mattermost server cannot connect to the database, Mattermost
+will fail to start. This section deals with MySQL database issues that
+you may encounter when you start up Mattermost for the first time.
+
+How you install MySQL varies depending upon which Linux distribution you
+use, but once MySQL is installed the configuration instructions are the
+same for all supported distributions. You must create a ``mattermost`` database
+and a ``mattermost`` database user. Failure to create these database
+objects or improperly referencing them from the Mattermost configuration
+file, ``/opt/mattermost/config/config.json``, prevents Mattermost from
```

Comment:
Same comment as above. We can probably just refer to this as config.json without the full path since we have the user edit it in the installation guides